eng/common/scripts/Verify-Links.ps1

```diff
@@ -14,7 +14,11 @@ param (
   # list of http status codes count as broken links. Defaults to 400, 401, 404, SocketError.HostNotFound = 11001, SocketError.NoData = 11004
   [array] $errorStatusCodes = @(400, 401, 404, 11001, 11004),
   # flag to allow resolving relative paths or not
-  [bool] $resolveRelativeLinks = $true
+  [bool] $resolveRelativeLinks = $true,
+  # development repo owner from pr
+  [string] $branchReplaceRegex = "",
```

Comment:
I think it would still be a good idea to have a regex defaulted here if for nothing more then to help document.